src/main/java/appeng/container/slot/SlotMACPattern.java

```diff
@@ -19,9 +19,8 @@
 package appeng.container.slot;
 
 
-import net.minecraft.inventory.IInventory;
 import net.minecraft.item.ItemStack;
-
+import net.minecraftforge.items.IItemHandlerModifiable;
 import appeng.container.implementations.ContainerMAC;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') 'appeng.container.implementations.ContainerMAC' should be separated from previous imports. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)